apps/firelens-stability/collections/ecs-firelens-stability-tests/golden-path/cases/no-onepod-high-throughput.json

```diff
@@ -0,0 +1,9 @@
+{
+    "definitions": {
+        "include_onepod": false,
+        "kbps_throughput_tcp": 5000,
+        "kbps_throughput_tail": 5000,
+        "kbps_throughput_forward": 5000,
+        "datajet_generator_basic_contentType": "uniform"
```

Comment:
is this the one we talked about which has an option to do random alphanumeric?

Reply:
Thank you for taking a look!

I have concerns with the performance with randomly generating a string for each log at high throughput. Not sure if the generator can keep up. For the uniform content, the same string is sent out multiple times, no new memory is generated.


With this change, we only use the alphanumerc/random for low throughput. The high throughput cases are set to uniform

Reply:
I see. Makes sense.